frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -154,7 +148,7 @@ export class VirtualMachine extends BaseVirtualMachine {
     if (startOnCreation) {
       await wizard.startOnCreation();
     }
-    await this.validateReviewTab(config, true);
+    await this.validateReviewTab(config);
```

Comment:
can you just do `this.validateReviewTab(config, false);` and leave the function above unchanged ?

Reply:
a I see this is the only usage ...